gcb/stage/cloudbuild.yaml

```diff
@@ -79,7 +79,7 @@ steps:
   - "--basedir=/workspace"
   - "--tmpdir=/workspace/tmp"
 
-- name: gcr.io/$PROJECT_ID/k8s-cloud-builder
+- name: gcr.io/k8s-staging-release-test/k8s-cloud-builder:${_KUBE_CROSS_VERSION}
```

Comment:
Following on https://github.com/kubernetes/release/pull/1141/files#discussion_r386089975, do we want to hard code the versions here as well? 🙂

Reply:
@saschagrunert -- Not yet. Since we're already using `_KUBE_CROSS_VERSION` in the config file, it's fine to continue using it here.

Reply:
SGTM